models/gc_nnunet_pancreas/config/default.yml

```diff
@@ -0,0 +1,35 @@
+general:
+  version: 1.0
+  data_base_dir: /app/data
+  description: base configuration for GC NNUnet Pancreas model (dicom to dicom)
+
+execute:
+- DicomImporter
+- MhaConverter
+- GCNNUnetPancreasRunner
+- DsegConverter
+- DataOrganizer
+
+modules:
+  DicomImporter:
+    source_dir: input_data
+    import_dir: sorted_data
+    sort_data: true
+    meta: 
+      mod: '%Modality'
+
+  MhaConverter:
+    engine: panimg
+    targets: [dicom:mod=ct]
+
+  DsegConverter:
+    model_name: 'GC NNUnet Pancreas'
+    source_segs: ['mha:mod=seg']
+    target_dicom: dicom:mod=ct
+    skip_empty_slices: True
+
+  DataOrganizer:
+    targets:
+    - mha:mod=heatmap-->[i:sid]/nnunet_pancreas_heatmap.mha
```

Comment:
please use the full model name as prefix for all generated output files,
e.g. `gc_nnunet_pancreas_heatmap.mha`.